eng/common/scripts/Collect-ChangeLogs.ps1

```diff
@@ -0,0 +1,60 @@
+[CmdletBinding()]
+param(
+  [Parameter(Mandatory=$true)]
+  [String] $FromDate # In the format "yyyy-MM-dd"
```

Comment:
Given you translate this string down lower, couldn't you just take a ```[DateTime]``` here instead of ```[String]```.